test/terraform/permanent/main.tf

```diff
@@ -1,13 +1,25 @@
-# read default vpc and create security group to validate basic setup and role permissions
-data "aws_vpc" "example_read_resource" {
+data "aws_vpc" "this" {
   id = "vpc-015d2f927c8b5dea7"
 }
 
-resource "aws_security_group" "example_write_resource" {
-  name   = "example_write_resource_from_terraform"
-  vpc_id = data.aws_vpc.example_read_resource.id
+data "aws_subnets" "this" {
+  filter {
+    name = "vpc-id"
+    values = [data.aws_vpc.this.id]
+  }
 
-  tags = {
-    created_by = "terraform"
+  filter {
+    name = "availability-zone"
+    values = ["us-east-1a", "us-east-1b"]
   }
-}
\ No newline at end of file
+}
+
+module "ci_e2e_cluster" {
+  source = "../modules/eks_cluster"
+
+  name       = "ci-e2e"
```

Comment:
This could be changed to `ci-e2etest` to match the nightly job, open to other ideas as well

Reply:
I think keeping it separate is a good idea so that we can also distinguish it easily in NR1. Might even be good to include `aws` or in the name to make that even more explicit